packages/instant/src/components/scaling_amount_input.tsx

```diff
@@ -71,6 +72,10 @@ export class ScalingAmountInput extends React.Component<ScalingAmountInputProps,
         );
     }
     private readonly _handleChange = (event: React.ChangeEvent<HTMLInputElement>): void => {
+        if (event.target.value === MAGIC_TRIGGER_ERROR_INPUT) {
+            throw new Error('Triggered error');
```

Comment:
Perhaps this message should be defined as a constant or as a `ZeroExInstantError` enumeration